### PR TITLE
Remove incorrect comment from MarkerLoggingAdapter and deprecate LogM…

### PR DESCRIPTION
**Pull request by jefftt**
addresses #24808

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @jefftt,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by jefftt** [comment]
Signed CLA

**Review by ktoso — APPROVED** [review]
Looks okey; the marker method isn't a direct replacement but it's a good enough suggestion

Thanks!

File: akka-actor/src/main/scala/akka/event/Logging.scala [hunk 1]
```diff
@@ -1441,6 +1441,8 @@ object LogMarker {
   /** Java API */
   def create(name: String): LogMarker = apply(name)
 
+  @Deprecated
+  @deprecated("use akka.event.LogEventWithMarker#marker instead", since = "2.5.12")
```
Comment (hepin1989) [inline akka-actor/src/main/scala/akka/event/Logging.scala]: `use` should be `Use`。

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**